readme.md

```diff
@@ -65,6 +66,7 @@ set --local command_x           \e'[O'
 set --local command_v           \e'[L'
 
 if functions --query _natural_selection
+  bind $escape              '_natural_selection end-selection'
```

Comment:
Which OS has this behaviour? I think this would be better at the end with an _Optional_ heading as macOS doesn't have this.

Reply:
I might be misunderstanding, but this is to allow the behavior that pretty much every IDE/text editor has where if you press escape with something selected it will end the selection.

Reply:
Yes, you're right. I didn't check the correct ones it turns out. Thanks.